hpe3parclient/client.py

```diff
@@ -2124,7 +2123,16 @@ def createCPG(self, name, optional=None):
         """
         info = {'name': name}
         if optional:
-            info = self._mergeDict(info, optional)
+           if self.primera_supported:
+               for key, value in dict(optional).items():
+                   if key == 'LDLayout':
+                       ldlayout=value
+                       for keys,val in dict(ldlayout).items():
+                           if keys == 'setSize':
```

Comment:
can we change the condition to below?
if keys == 'setSize' || (keys == 'RAIDType' and ldlayout.get('RAIDType')==1):

Reply:
Addressed review comments